### PR TITLE
tiekinetix/index.hbs

### PR DESCRIPTION
**Pull request by WernerDH**
@teeyoutan creating new partner directory entry

### CODE REVIEW

**Comment by dtothefp** [comment]
@teeyoutan you check this out?

File: website/partners/solutions/tiekinetix/index.hbs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: partners.hbs
+title: "TIE Kinetix"
+solutions: true
+priority: 50
+summary:
+logo: "//optimizely-marketer-assets.s3.amazonaws.com/partners/logos/tiekinetix.png"
+website_link: "http://tiekinetix.com/de/optimizely-tiekinetix-websites-optimieren"
```
Comment (dtothefp) [inline website/partners/solutions/tiekinetix/index.hbs]: @teeyoutan @WernerDH something is wrong with the logo link. Did you actually upload the logo to Cloud Cannon?  The folder should be `/partners-logos/solutions/<filename>`

screen shot 2015-03-05 at 4 01 25 pm

**Comment by dtothefp** [comment]
@WernerDH I just added you to cloud cannon access where we store the images

screen shot 2015-03-05 at 4 04 42 pm

**Comment by WernerDH** [comment]
Thanks David, I assume I am now able to add the Tie Kinetix logo trough
clod cannon?
Am 05.03.2015 22:04 schrieb "David Fox-Powell" <notifications@github.com>:

**Comment by dtothefp** [comment]
@WernerDH yes you should have received an email from them, if not go to cloud cannon and try to login

**Comment by teeyoutan** [comment]
@julieritchie  Can you look into this one?

**Comment by julieritchie** [comment]
@dtothefp I don't think that partner managers are resizing partner logos in photoshop before adding them to Cloud Cannon.

**Comment by julieritchie** [comment]
@teeyoutan yes, of course I will look into this :)

**Comment by dtothefp** [comment]
retest this please

**Comment by WernerDH** [comment]
@stewsmith : hey stewart, could you take a look at this PR, as far as I can see it should be alright.

**Comment by julieritchie** [comment]
@WernerDH @stewsmith I'll take a look at this. Werner please don't pull in Stew or DFP until I've reviewed it as it increases cycle time

**Comment by stewsmith** [comment]
@julieritchie @WernerDH Please let me know when you'd like review.

**Comment by julieritchie** [comment]
Thanks @stewsmith I'll get to this tomorrow. Going to use it to train Kelly

**Comment by stewsmith** [comment]
@julieritchie Just added a commit to fix the merge conflict.

**Comment by stewsmith** [comment]
@julieritchie Please let me know when this is ready to merge. Thanks!

**Comment by kylerush** [comment]
@julieritchie can we get an update on this por favor?

**Comment by WernerDH** [comment]
@julieritchie can we take this PR live?

**Comment by julieritchie** [comment]
@WernerDH sorry for the delay. We will for sure get this done this week! I have a contractor starting this week to help work on this stuff and she has not been given access to GitHub this week, hence the delay. But it is on my radar and will have it done by end of the week at the latest

**Comment by julieritchie** [comment]
@stewsmith this is ready.

**Comment by stewsmith** [comment]
LGTM

**Comment by stewsmith** [comment]
@julieritchie Thanks!

**Pull request merged**